diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/FileStructureRuleTest.kt

```diff
@@ -327,4 +327,54 @@ class FileStructureRuleTest : LintTestBase(::FileStructureRule) {
             """.trimMargin(),
         )
     }
+
+    @Test
+    @Tag(WarningNames.UNUSED_IMPORT)
+    fun `Operator overloading 2`() {
```

Comment:
```suggestion
    fun `Should correctly check infix functions`() {
```
Please also add a negative example - when infix function is imported and unused, there should be a warning.